druid/src/main/java/org/apache/calcite/adapter/druid/DruidQuery.java

```diff
@@ -833,6 +863,54 @@ protected JsonAggregation getJsonAggregation(List<String> fieldNames,
     }
   }
 
+  public JsonPostAggregation getJsonPostAggregation(String name, RexNode rexNode, RelNode rel) {
+    if (rexNode instanceof RexCall) {
+      List<JsonPostAggregation> fields = new ArrayList<>();
+      for (RexNode ele : ((RexCall) rexNode).getOperands()) {
+        JsonPostAggregation field = getJsonPostAggregation("", ele, rel);
+        if (field == null) {
+          return null;
+        }
+        fields.add(field);
+      }
+      switch (rexNode.getKind()) {
+      case PLUS:
+        return new JsonArithmetic(name, "+", fields, null);
+      case MINUS:
+        return new JsonArithmetic(name, "-", fields, null);
+      case DIVIDE:
+        return new JsonArithmetic(name, "quotient", fields, null);
+      case TIMES:
+        return new JsonArithmetic(name, "*", fields, null);
+      case CAST:
+        return getJsonPostAggregation(name, ((RexCall) rexNode).getOperands().get(0),
+            rel);
+      default:
+      }
+    } else if (rexNode instanceof RexInputRef) {
+      if (rel instanceof Aggregate) {
+        Integer indexSkipGroup = ((RexInputRef) rexNode).getIndex()
+            - ((Aggregate) rel).getGroupSet().cardinality();
```

Comment:
Sure, I was following the code [here](https://github.com/apache/calcite/blob/56d5261abb893dd92b0a2cc4893292876a2b7880/core/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java#L2760) when I am reading the codebase.